presto-kafka/src/test/java/io/prestosql/plugin/kafka/KafkaQueryRunner.java

```diff
@@ -65,7 +65,9 @@ static DistributedQueryRunner createKafkaQueryRunner(TestingKafka testingKafka,
     {
```

Comment:
this is going to be covered by https://github.com/prestosql/presto/pull/2164

Also commit message is wrong.`KafkaQueryRunner` is not deprecated.